README.md

```diff
@@ -49,12 +49,12 @@ Add this to your `Cargo.toml`:
 c2pa = "0.11.3"
 ```
 
-If you want to read or write a manifest file, add the `file_io` dependency to your `Cargo.toml`. For example:
+If you want to read or write a manifest file, add the `file_io` feature to your `Cargo.toml`. For example:
 ```
 c2pa = { version = "0.11.0", features = ["file_io"] }
 ```
 
-NOTE: If you are building for WASM, omit the `file_io` dependency.
+NOTE: If you are building for WASM, you can not use the `file_io` feature. (It relies on other crates that are not currently available on WASM.)
```

Comment:
NOTE: If you are building for Wasm, you cannot use the `file_io` feature since it relies on other crates not currently available on Wasm.

(spelling/capitalization)